code/_onclick/hud/hud.dm

```diff
@@ -205,6 +205,83 @@ datum/hud/New(mob/owner)
 	else if(isguardian(mymob))
 		guardian_hud()
 
+	create_parallax()
+
+
+/client/var/list/obj/screen/spessbg/spessbg = list()
```

Comment:
Lists cannot have types, and defining them with such makes it very difficult to quickly find something in the code, `grep -rn 'client/var/list/spacebg' code/` won't return anything. So this should just be `/client/var/list/spessbg = list()`